src/widget.rs

```diff
@@ -89,8 +89,21 @@ impl<'a> Renderer<'a> {
         let lnum_len = num_digits(lines_len);
         let bottom_row = cmp::min(top_row + height, lines_len);
         let mut lines = Vec::with_capacity(bottom_row - top_row);
+        // big enough for the largest line I can imagine
+        const STARS: &str = "******************************\
```

Comment:
This is probably not a good idea as things can be longer than you'd imagine ;)

Reply:
its per line. And thats 180 chars. I could not imagine a line longer than that, the UI still works in that case.

Reply:
placeholder works fine (I admit i did not test it till just now , which is dumb since i wrote the placeholder feature)